docs/source/UserGuide/GettingStarted/GettingUserIdentity/index.rst

```diff
@@ -61,3 +61,20 @@ If another non-default user group is needed, the command becomes::
   $ dirac-proxy-init -g <user_group>
 
 where ``user_group`` is the desired DIRAC group name for which the user is entitled.
+
+Token authorization
+-------------------
+
+Starting with the 8.0 version of DIRAC, it is possible to authorize users through third party Identity Providers (IdP),
+such as EGI Checkin [https://www.egi.eu/services/check-in/] or WLCG IAM (https://indico.cern.ch/event/739896/contributions/3497694/attachments/1905332/3146590/IAM-WLCG-AuthZ-Fermilab-10092019.pdf).
```

Comment:
Please use proper rst links. 
For IaM maybe https://github.com/indigo-iam/iam or https://indigo-iam.github.io/v/current/ are better links